### ISSUE TITLE
Overview?

### ISSUE THREAD

**Issue by richardmadison** [electroneum/electroneum-pool#16]
Do we need to be running a wallet script on the pool server for this to work?

We're running electroneumd, redis-server and node init.js.

Just curious because it appears to work without any wallet script running. The docs don't mention running a wallet, but other forks of node-cryptonote-pool mention simplewallet.

**Comment by TobCar** [comment]
We are running electroneum-wallet-rpc, our pool outputted errors otherwise. It's probably just an oversight in the README, the Electroneum team is overworked right now. (But they're doing a great job!)

**Comment by richardmadison** [comment]
If it's not too much trouble, can you let me know what cmd you are running for your wallet?

**Comment by TobCar** [comment]
No problem! This took me at least an hour to figure out, maybe it will help you too.

`electroneum-wallet-rpc --wallet-file [Pool wallet file including the .etn] --rpc-bind-port 18081`

You will need to generate your wallet file using electroneum-wallet-cli as if you were importing a paper wallet for the first time. Secondly, make sure the RPC port you are binding to is the same RPC port you will use everywhere else.

**Comment by richardmadison** [comment]
@TobCar - Thanks. Another quick question...

**Comment by TobCar** [comment]
We actually left that alone. We don't know why but our pool wouldn't launch otherwise. We did change the daemon port to the same port as electroneumd was using for rpc. Intuitively, this should have meant we wouldn't have to run electroneum-wallet-rpc separately since electroneumd was clearly connecting to that port anyway, but electroneumd threw an error if we didn't run the wallet at the same time.

Here is the command we used to change the ports on electroneumd:
`electroneumd --rpc-bind-port 26968 --p2p-bind-port 26967 --detach`

And here is our config file:
`"daemon": {
        "host": "127.0.0.1",
        "port": 26968
    },`

Perhaps a dev can chime in as to why electroneum-wallet-rpc has to run even though it is on a different port.

Side note: **Set a password on Redis.** It is very easy to exploit if you don't.
